pkg/pillar/hypervisor/xen.go

```diff
@@ -195,6 +195,9 @@ func (ctx xenContext) CreateDomConfig(domainName string, config types.DomainConf
 		if ds.Format == zconfig.Format_CONTAINER {
 			p9Strings = append(p9Strings,
 				fmt.Sprintf("'tag=share_dir,security_model=none,path=%s'", ds.FileLocation))
+		} else if ds.IsBlank {
```

Comment:
Is `IsBlank` the right thing to do here? The hypervisor doesn't care if it is _blank_ or not, but rather if it is a _directory_ or not. Maybe this whole `IsBlank` should be changed to `IsDirectory`, and if it is a directory, then it needs p9 mounting?

Reply:
Makes sense. Let me change it.